src/app/content/components/Content.tsx

```diff
@@ -60,37 +57,6 @@ const ContentNotifications = styled(Notifications)`
   }
 `;
 
-// tslint:disable-next-line:variable-name
-const UndoPadding = styled.div`
-  @media screen {
-    overflow: visible;
-    min-height: 100%;
-    display: flex;
-    flex-direction: column;
-  }
-`;
-
-// tslint:disable-next-line:variable-name
-const MainContentWrapper = isVerticalNavOpenConnector(styled.div`
-  @media screen {
-    flex: 1;
-    display: flex;
-    flex-direction: column;
-    overflow: visible;
-    transition: max-width ${sidebarTransitionTime}ms;
-    width: 100%;
-    margin: 0 auto;
-    padding-left: 0;
-    ${theme.breakpoints.mobile(css`
-      padding-left: 0;
-      max-width: ${contentWrapperMaxWidth}rem;
-    `)}
-    ${(props) => (props.isVerticalNavOpen === false) && `
-      max-width: ${contentWrapperMaxWidth}rem;
-    `}
-  }
-`);
```

Comment:
These containers did not see to be doing anything. `UndoPadding` used to have some negative margin logic but that was removed when the new nav was implemented. I haven't found any visual regressions after removing these.